src/validation.cpp

```diff
@@ -310,6 +310,9 @@ bool CheckSequenceLocks(const CTransaction &tx, int flags, LockPoints* lp, bool
 }
 
 
+// Returns the script flags which should be checked for a given block
```

Comment:
I'm not following, in this commit this additional declaration (when nobody is calling it between the declaration and the definition) is completely redundant. The definition is close to ConnectBlock, also consensus code.